### DISCUSSION TITLE
react-native-reanimated has peerDependencies

### DISCUSSION THREAD

**Discussion by samuel-rl** [software-mansion/react-native-reanimated#2082] [Q&A]
Hello, I have installed react-native-reanimated ^2.0.0 (previously react-native-reanimated ^1.3.0 on this project).

Actually, I need 1.3.0 has peerDependencies for  `react-navigation-drawer": "^2.2.2"`.

I got this, maybe because the drawer use reanimated v2 instead of v1 ?
<img width="444" alt="Capture d’écran 2021-06-01 à 10 10 04" src="https://user-images.githubusercontent.com/44348413/120289832-00ee6700-c2c2-11eb-9884-6fdf5ac66a91.png">

This is not possible to have 2 versions of reanimated installed even if it's specify in `package.json` in peerDependencies ?

My `package.json`:

```
 "dependencies": {
    "react-native-reanimated": "^2.0.0",
    "react-navigation-drawer": "^2.2.2",
  },
```

**Comment by jgonet** [comment]
You can use [yarn resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/#toc-how-to-use-it) to force using ^2.0.0. Keep in mind Reanimated 2 will issue warnings about drawer using `interpolate()` instead of `interpolateNode()`. I explained in [this comment](https://github.com/software-mansion/react-native-reanimated/discussions/2074#discussioncomment-806970) how to suppress that warning.

**Reply by samuel-rl** [reply]
But actually, I got this error, Drawer.js look to use the v2 instead of the v1, so it don't find interpolate function. This is not warning.

<img width="444" alt="Capture d’écran 2021-06-01 à 10 10 04" src="https://user-images.githubusercontent.com/44348413/120289495-a523de00-c2c1-11eb-8b73-b58664c493b8.png">

**Reply by jgonet** [reply]
`interpolate` in this form is an old `interpolate` from v1. Drawer imports it from `Animated` object [here](https://github.com/react-navigation/react-navigation/blob/9c75df13c1d1b37fbb76179b09ddb988b2da8b68/packages/drawer/src/views/Drawer.tsx#L25) and we stopped exporting Reanimated v2 functions inside this object at [2.2.0 version](https://github.com/software-mansion/react-native-reanimated/commit/2a049a01f2ae7795d16ce28fbf785429f340baec). Try resolving Reanimated to version "2.1.0" instead of "^2.0.0".
